src/filters/detector.cpp

```diff
@@ -55,8 +81,15 @@ Try<Nothing> DetectorFilter::consume(const ResourceUsage& in) {
           continue;
         }
 
-        // Detected change point.
+        // Detected contention.
         if (cpDetected.isSome()) {
+          // Check if aggressors jobs are available on host.
+          if (currentAggressors.executors_size() == 0) {
+            SERENITY_LOG(INFO) << "Contention spotted, however there are no "
+                << "aggressor on the host. Assuming false positive.";
```

Comment:
We should rather inform, that agressors are currenty best effort jobs.

:s/no agressors/no best efforts.

We might also use this notation in code, to reduce confusion in readers.